src/backend/marsha/settings.py

```diff
@@ -18,6 +18,8 @@
 
 BASE_DIR = os.path.dirname(os.path.abspath(__file__))
 
+# pylint: disable=abstract-class-instantiated
```

Comment:
Move to the top!

https://pylint.pycqa.org/en/latest/faq.html#is-there-a-way-to-disable-a-message-for-a-particular-module-only

Reply:
Done